### PR TITLE
Pal 1037 palisade service readme

### PR DESCRIPTION
**Pull request by pd104923**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ First it will return a reference, a token back to the client that will uniquely
 This token is used throughout the application to tie every aspect of the process to this unique identifier and is later used by the client to retrieve the resources available for this request. 
 The second task for the Palisade Service is to initiate the processing of the request by forwarding it onto the next service in the sequence (in this case the User Service).
 The service will perform its required task and then forward the message onto the next service with the end goal of producing resources ready to be viewed by the client.
-This sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed data.
+This sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed request.
```
Comment (m78233) [inline README.md:32]: Late in the day, but may be relevant.  In the documentation for Filtered Resource Service there was also the use of resource in the same context, e.g. "The Filtered-Resource Service sits at the end of the stream pipeline and has the task of returning resources (and their connection details) back to the client. "  This give the idea that there is more than data.

File: policy-service/README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ If no record or resource level rules are found for this resource, a `NoSuchPolic
 An `AuditErrorMessage` is then built, containing the user information, context, resourceId, the error thrown, and any additional attributes. 
 This error object is then sent with the `PolicyRequest` to the [Audit service](../audit-service) to be audited appropriately.
 If, however, resource level rules are found, and when applied against the resource, the resource has not been redacted, then the Policy Service will check the cache for Record Level rules. 
-Record Level Rules are rules that can be applied against the data itself, and based off context can limit what the user can see in a particular resource.
+Record Level rules are rules that can be applied against the data itself, and based off context can limit what the user can see in a particular resource.
```
Comment (m78233) [inline policy-service/README.md]: Rules was capitalised because it refers to the Rules class.

File: services-manager/README.md [hunk 3]
```diff
@@ -21,14 +21,16 @@ The source for this file can be found at `services-manager/src/resources/doc/REA
 
 # <img src="../logos/logo.svg" width="180">
 
-## A Tool for Complex and Scalable Data Access Policy Enforcement
+### Palisade Services Manager
```
Comment (m78233) [inline services-manager/README.md]: Would have expected the line A Tool for Complex...  to be there.

**Review by m78233 — COMMENTED** [review]
A few minor comments.

File: policy-service/README.md (hunk 2 above)
Comment (pd104923) [reply policy-service/README.md]: Okay, I will revert this

File: services-manager/README.md (hunk 3 above)
Comment (pd104923) [reply services-manager/README.md]: Added this back in

File: README.md (hunk 1 above)
Comment (pd104923) [reply README.md:32]: Hmm, good spot. Made a small change within the FRS ReadMe file

**Review by m78233 — APPROVED** [review]
Looks good

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**